debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/TopicSelectorTest.java

```diff
@@ -30,8 +30,14 @@ public void beforeEach() {
 
     @Test
     public void shouldHandleCollectionIdWithDatabaseAndCollection() {
-        assertTopic(noPrefix,dbAndCollection("db", "coll")).isEqualTo("db.coll");
-        assertTopic(withPrefix,dbAndCollection("db", "coll")).isEqualTo("prefix.db.coll");
+        assertTopic(noPrefix, dbAndCollection("db", "coll")).isEqualTo("db.coll");
+        assertTopic(withPrefix, dbAndCollection("db", "coll")).isEqualTo("prefix.db.coll");
+    }
+
+    @Test
```

Comment:
I think this should also contain `@FixFor`

Reply:
Good idea.